cargo-bolero/src/project.rs

```diff
@@ -23,9 +23,11 @@ pub struct Project {
     #[structopt(long)]
     all_features: bool,
 
-    /// Build artifacts in release mode, with optimizations [default: true]
-    #[structopt(long)]
-    release: Option<Option<bool>>,
+    /// Build artifacts in release mode, with optimizations [default: "fuzz"]
+    ///
+    /// Note that if you do not have `codegen-units = 1` in the profile, there are known compilation bugs
+    #[structopt(long, default_value = "fuzz")]
```

Comment:
Should we default to a profile that is there by default?

Reply:
So my thinking was, it’s better to fail loudly than to fail silently.

If we default to release, then we open ourselves up to the LTO issue you mentioned above, and there won’t be debug-assertions. If we default to debug, then the fuzzer will be way too slow for reasonable running.

Right now, the error message for people who didn’t read through the book would be:
```
error: profile `fuzz` is not defined
```
I think it’s reasonable that the user would search the internet with this error message, and fall on the library-installation.md page (to which I just added this error message, to make sure it’s easy to find).

We could also add a log message at the beginning of each cargo-bolero invocation, reminding to read the book if it fails with this error message, but that’d feel overkill to me.

WDYT?

Reply:
Just did it! See #168 :)